### DISCUSSION TITLE
dispatchTo from livewire component is not available ?

### DISCUSSION THREAD

**Discussion by invaders-xx** [livewire/livewire#8037] [1. Help] [answered]
It there a way to send an event to a specific livewire component within another livewire component ?

**Comment by danie-ramdhani** [comment, marked as answer]
chain it with `to` function (https://livewire.laravel.com/docs/events#dispatching-directly-to-another-component):

```php
$this->dispatch('post-created')->to(Dashboard::class);
```

**Reply by invaders-xx** [reply]
Many thanks!
